<launch>
  <arg name="device" default="/dev/ttyUSB0" />
  <arg name="rate" default="1000" />
  <arg name="sensor_frame_id" default="/sensor" />
  <arg name="topic" default="/force" />
  <arg name="frequency_div" default="1" />
  <arg name="acquire_calibration" default="true"/>

  <node pkg="dynpick_driver" name="dynpick_driver_node" type="dynpick_driver_node" >
    <param name="device" value="$(arg device)" />
    <param name="rate" value="$(arg rate)" />
    <param name="frame_id" value="$(arg sensor_frame_id)" />
    <param name="frequency_div" value="$(arg frequency_div)" />
    <param name="acquire_calibration" value="$(arg acquire_calibration)"/>
    <remap from="/force" to="$(arg topic)" />
  </node>
</launch>
